_maps/map_files/Delta/delta.dmm

```diff
@@ -34368,7 +34368,7 @@
 	dir = 4
 	},
 /turf/space,
-/area/space/nearstation)
+/area/space)
```

Comment:
`/area/space/nearstation` does not appear to be in

Reply:
@Crazylemon64 
![image](https://user-images.githubusercontent.com/25063394/57579809-51a38500-7499-11e9-96b6-3ad03898d451.png)
It is in, that may just be an area that I tweaked